indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,265 @@
+        <div class="description">
+            <span class="bold">{% trans %}Description{% endtrans %}</span>
+            <div>{{ contrib.description }}</div>
+        </div>
+    {% endif %}
+    {% if contrib.subcontributions %}
+        <div>
+            <span class="bold">{% trans %}Subcontributions{% endtrans %}</span>
+            <ul class="sub-contrib-timeline inherited-bg-color-trans inherited-border-color">
+                {% for subchild in contrib.subcontributions %}
+                    <li class="inherited-border-color">
+                        <div class="contrib-heading">
+                            {% if show_length_contribs and not contrib.session.is_poster %}
+                                <span class="timebox inherited-bg-color inherited-txt-color">
+                                    {{ (subchild.duration)|format_human_timedelta(narrow=True) }}
+                                </span>
+                            {% endif %}
+                            <h4>{{ subchild.title }}</h4>
+                        </div>
+                        <div class="contrib-timeline-content">
+                            {{ render_nested_contrib(subchild, show_abstract, dont_show_poster_abstract, show_title, show_affiliation, show_length_contribs) }}
+                        </div>
+                    </li>
+                {% endfor%}
+            </ul>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_block_date(entry, day, print_date_close_to_sessions) %}
+    {% if print_date_close_to_sessions %}
+        <p class="session-block-date">
+            <span class="bold">{{ day|format_date('EEEE, d MMMM', timezone=entry.event.tz) }}</span>
+            <span class="timebox">
+                {{ format_interval(entry.start_dt.astimezone(entry.event.tzinfo), entry.end_dt.astimezone(entry.event.tzinfo), format='hh:mm') }}
+            </span>
+        </p>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_time(entry) %}
+    <div class="time">
+        <div class="time-from">
+            {{ entry.start_dt|format_time(timezone=entry.event.tz) }}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-to">
+            {{ entry.end_dt|format_time(timezone=entry.event.tz) }}
+        </div>
+    </div>
+{% endmacro %}
+
+{# Utilities #}
+
+{% macro render_place(entry) %}
+    {% set entry = entry.object if entry.object else entry %}
+
+    {{ [entry.venue_name, entry.room_name, entry.address] | select | join(', ') }}
+{% endmacro %}
+
+{% macro render_names(people, show_title=false, show_affiliation=false) %}
+    {% for person in people %}
+        {{ person.get_full_name(show_title=show_title, abbrev_first_name=false, last_name_first=false, last_name_upper=false) }}
```

Comment:
I think `display_full_name` might be better here. It takes the event's `NameFormat` into account. The only issue is you can't control the title, but maybe we could add it as an extra parameter?

Reply:
That's better and yes let's add the title.